stl/inc/xutility

```diff
@@ -3584,7 +3601,9 @@ namespace ranges {
             auto& _Self = _Cast();
 #if _CONTAINER_DEBUG_LEVEL > 0
             if constexpr (sized_range<_Derived>) {
-                _STL_VERIFY(_Idx < _RANGES size(_Self), "index out of range for view_interface");
+                using _Ty = _Make_unsigned_like_t<range_difference_t<_Rng>>;

```

Comment:
As far as I can tell this is conventionally called `_Diff`

Reply:
We use `_Diff` pretty exclusively for difference types - I think there may be a `_UDiff` somewhere for "unsigned version of difference type"?

Reply:
Sort of:

https://github.com/microsoft/STL/blob/484fbc9742f895030047977efc55dc8950a37086/stl/inc/xutility#L5818-L5821

I'll use `_U_diff` which I think is more consistent with our modern naming conventions.